### PR TITLE
Feature/breadcrumbs

### LINKED ISSUE #17: Component: Breadcrumbs
Create the "breadcrumbs" component that will be included in the site header.
screen shot 2018-10-23 at 12 42 50 pm

- [ ] Collection Title
- [ ] Brief Type (only for Issue Briefs on single posts)
- [ ] Post Title (only on single posts)

### PR DESCRIPTION
**Pull request by aleesteele**
closes #17 
- Individual pages display breadcrumb: for issues: 'Issue Brief / Brief Type / Brief Name', for analysis/posts: 'Analysis / Analysis Name', etc.
- Archive pages display as "Home / Archive Page"
- Individual Authors posts display as "Analysis / Author Name"
Individual posts look good to me, although for Authors, I would actually link it to Analysis instead of Authors, like so "Analysis / Author Name"

### CODE REVIEW

File: _includes/breadcrumbs.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  {% if page.collection == 'resources' or page.collection == 'workshops' %}
+    {% assign breadcrumb_home = page.type %}
+  {% else %}
+    {% assign breadcrumb_home_path = site.collections | where: "label", page.collection | first %}
+    {% assign breadcrumb_home = breadcrumb_home_path.title %}
+  {% endif %}
+
+  {% if page.collection == 'briefs' %}
+    {% assign breadcrumb_past = page.brief_type %}
+  {% else %}
```
Comment (jnschrag) [inline _includes/breadcrumbs.html]: The else part of this statement isn't necessary since breadcrumb_past was already set to null above.

Comment (aleesteele) [reply _includes/breadcrumbs.html]: You're right, deleted this!

**Comment by aleesteele** [comment]
- used `https://ontheradar.csis.com/` as site.url, but removed before pushing this.
- added links to the previous breadcrumbs
- renamed breadcrumb_home to breadcrumb_first

**Comment by aleesteele** [comment]
Fixed, @jnschrag @leezaweb.
- Removed the `site.url | append:` logic
- The URLs were like that for 'Home' because there's not currently a `site.url` value. For now, I assigned breadcrumb_first_url to `'/'`, but this will have to be assigned to site.url when the url is up and running. Apologies, I explained this terribly in the above comment.
-   Fixed the breadcrumbs for workshops as well my if statement logic was extraneous.

**Comment by jnschrag** [comment]
@aleesteele Can you go ahead and set the url in `_config.yml` since it's required for this to work? You can set it to https://ontheradar.csis.org. Let me know when that's done and I'll review the rest of it.

**Comment by aleesteele** [comment]
@jnschrag added the url to the config file! :)

**Comment by jnschrag** [comment]
@aleesteele Thanks for fixing. The first link when viewing the individual Analysis, Issue Briefs, and Workshops posts is incorrect. Eg, the Analysis link is trying to direct the user to `/Analysis/` instead of `/analysis` and `/Issue Briefs/` instead of `/issue-briefs/`.

**Comment by aleesteele** [comment]
@jnschrag @leezaweb 

This has been fixed by slugifying the url. Just a note however, I was playing around the search settings, and I think Algolia might be set up to search using the non-slugified search terms as well. I attached a screen shot example. I'm not sure if this is case though.
screen shot 2018-11-29 at 9 17 36 am

**Pull request merged**